src/examples/pane/PaneVertical.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { PaneProvider, Pane } from '@zendeskgarden/react-grid';
+import useResizeObserver from 'use-resize-observer';
+import { XL, Paragraph } from '@zendeskgarden/react-typography';
+import { PALETTE } from '@zendeskgarden/react-theming';
+import styled from 'styled-components';
+
+const StyledPaneContent = styled(Pane.Content)`
+  padding: ${p => p.theme.space.base * 6}px;
+  height: 240px;
+`;
+
+const StyledParagraph = styled(Paragraph)`
+  margin-top: ${p => p.theme.space.xs};
+`;
+
+const Example = () => {
+  const { ref, width = 1, height = 1 } = useResizeObserver();
+
+  return (
+    <div ref={ref}>
+      <PaneProvider
+        totalPanesHeight={height}
+        totalPanesWidth={width}
+        defaultColumnValues={{
+          'column-1': 1,
+          'column-2': 1
+        }}
+      >
+        {({ getGridTemplateColumns }) => (
+          <div
+            style={{
+              width,
+              height: '100%',
+              display: 'grid',
+              gridTemplateColumns: getGridTemplateColumns()
+            }}
+          >
+            <Pane>
+              <StyledPaneContent
+                style={{
+                  backgroundColor: PALETTE.grey[100]
+                }}
```

Comment:
[nit] ditto.

Reply:
Updated in https://github.com/zendeskgarden/website/pull/416/commits/f00aab8b4acb4eb8cd3dc995da1153e5fc5c10a3.